resources/codegen_inputs/trak.rs

```diff
@@ -0,0 +1,43 @@
+#![parse_module(read_fonts::tables::trak)]
+
+/// The [tracking (trak)](https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6trak.html) table.
+#[tag = "trak"]
+table Trak {
+    /// Version number of the tracking table (0x00010000 for the current version).
+    version: MajorMinor,
+    /// Format of the tracking table (set to 0).
+    format: u16,
```

Comment:
might as well also add `#[compile(0)]` here? (and `#[compile(MajorMinor::VERSION_1_0)]` above?)

Reply:
will do!